config/pipeline.yaml

```diff
@@ -908,6 +908,162 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.iommu
 
+  ltp: &ltp
+    template: ltp.jinja2
+    kind: job
+    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-ltp/20240313.0/{debarch}'
+    params: &ltp-params
+      skip_install: "true"
+      tst_cmdfiles: ""
```

Comment:
This should be dropped (see review in core PR)

Reply:
Dropped from here.